contracts/VeJoeStaking.sol

```diff
@@ -21,14 +21,7 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
 
     /// @notice Info for each user
     /// `balance`: Amount of JOE currently staked by user
-    /// `lastRewardTimestamp`: Latest timestamp that user performed one of the following actions:
-    ///     1. Claimed pending veJOE
-    ///     2. Staked JOE with zero balance
-    ///     3. Staked JOE with non-zero balance and is at their max veJOE cap after claim
-    ///     4. Unstaked JOE
-    /// `boostEndTimestamp`: Timestamp of when user stops receiving boost benefits. Note that
-    /// this will be reset to 0 after the end of a boost
-    /// `rewardDebt` is the reward debt of the user. Doesn't take into account the boosted generation.
+    /// `rewardDebt`: The amount of rewards user has already claimed
```

Comment:
This is not really an accurate description of the `rewardDebt`...
Tho I can't find a good one x)